### PR TITLE
mainly adding a page of inspirations

### PR DESCRIPTION
**Pull request by dougli1sqrd**
@sethp  Do you have any other inspirations that you'd want to add?

### CODE REVIEW

File: riscv-router.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ Selling a low-cost, low-power, small (2-6 port) gigabit router that's easy to ha
 - operations / legal
 - [licensing](./licensing.md)
 
+Obviously it should spit out UART somewhere so we can plug Vgaterm into it.
```
Comment (sethp) [inline riscv-router.md:15]: :laughing:  Good news: a serial console is absolutely A Thing that we're gonna want to figure out for this device as a separate access path if the network bits get borked.

File: inspirations.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Projects and Orgs that Inspire Rustbox
+
+* Oxide
+* Sparkfun
```
Comment (sethp) [inline inspirations.md:4]:
```suggestion
* Adafruit
* Sparkfun
```

File: inspirations.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+* [Scenic Routes](https://www.scenicroutessf.com/)
+* https://ploopy.co/about-us/
+* [Justine Haupt's Rotary Cell Phone](https://justine-haupt.com/rotarycellphone/)
+  * [Sky's Edge](https://www.skysedge.com/philosophy/index.html)
+* Fairphone
+* Pine Phone
+* [DevTerm](https://www.clockworkpi.com/devterm) and ClockworkPi
+* [Watchy](https://watchy.sqfmi.com/)
+* [Open Pick and Place by Opulo](https://www.youtube.com/@stephen_hawes)
```
Comment (sethp) [inline inspirations.md:13]:
```suggestion
* [Open Pick and Place by Opulo](https://www.youtube.com/@stephen_hawes)
* open collective 
```

**Review by sethp — APPROVED** [review]

**Comment by sethp** [comment]
Also, not sure where to put it, but I feel some affinity for the goals of the folks over at https://www.servethehome.com/ too.

**Comment by dougli1sqrd** [comment]
oh i'm not familiar with serve the home! What do you like about them? But if you're inspired you should add it :smile:

**Comment by sethp** [comment]
They do reviews & run a forum for people that sit on the boundary between "(expert) home user" and "small (home) lab operator".

I don't know much about them besides that, but if we do manage to build a widget, they seem like the sort of folks that might be interested in it. 

In other words: once we get something working, floating it around over there might get us some potential "late alpha" or "early beta" interest.

File: inspirations.md (hunk 2 above)
Comment (sethp) [reply inspirations.md:4]: Oh, obviously also OSH Park, right?

**Comment by sethp** [comment]
Oh, and https://turingpi.com/ is kind of an interesting effort to me too. Does that match what you're thinking for "inspiration"?

**Comment by dougli1sqrd** [comment]
Ha yeah that seems good! Lol my main suspicion of them is  :ghost:  "AI inference at the edge" :ghost: 

*Open and Close ghost emojis clearly indicate reading enclosed text in a spooky voice*

**Pull request merged**